cvat-ui/src/reducers/selection-reducer.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { SelectionState } from 'reducers';
+import {
+    SelectionActionsTypes,
+    SelectionActions,
+} from '../actions/selection-actions';
+
+const initialState: SelectionState = {
+    selected: [],
+    fetching: false,
+    status: null,
+    cancelled: false,
+};
+
+export default function selectionReducer(
```

Comment:
As mentioned before, each state (Projects/Tasks ... etc) should store selected items. 
In this case this one would be `bulkOperationReducer`. 

Select/Deselect may be a separated ActionTypes supported by different reducers in common code.